T13980

rshuler opened the pull request:
https://phabricator.endlessm.com/T13980

Commit:
Script to install google-chrome

Downloads the debian package from Google, unzips to the user's
home directory, and creates a desktop launcher.

Note that libgconf-2-4 must be installed for google-chrome to run.

https://phabricator.endlessm.com/T13980

Commit:
Bundle libgconf-2-4 (from Debian Stable) as a dependency for Chrome

https://phabricator.endlessm.com/T13980

Commit:
Refactor the code a bit now the script is downloading two packages

Also, run update-desktop-database after installing the desktop file.

https://phabricator.endlessm.com/T13980

cosimoc commented on eos-install-google-chrome:
```diff
@@ -0,0 +1,154 @@
+esac
+
+TEMP_DIR=`mktemp -d`
+DEST_DIR=~/.google-chrome
```
As commented on the older PR, please use a different location for this.

mariospr commented on eos-install-google-chrome:
```diff
@@ -0,0 +1,154 @@
+# Downloads Google Chrome and unzips to the user's home directory,
+# with a desktop launcher
+
+exit_with_error() {
```
As for the case with the Firefox script, it would be a good idea to make sure that ${TEMP_DIR} gets removed whenever this function is called. Sorry I did not notice this before, when writing my patches on top of yours.

Commit:
eos-install-google-chrome: minor refactor

Make sure the temporary directory is cleaned up on exit,
and keep the add-to-desktop functionality in one place.

https://phabricator.endlessm.com/T13980

Commit:
eos-install-google-chrome: provide icon override

Using the upstream icon with a transparent background looks odd
with our rounded icon well, so we resize it a little and provide
a solid background.

https://phabricator.endlessm.com/T13980
